pkg/tinydtls/contrib/sock_dtls.c

```diff
@@ -427,7 +418,6 @@ ssize_t sock_dtls_recv(sock_dtls_t *sock, sock_dtls_session_t *remote,
         if (mbox_try_get(&sock->mbox, &msg)) {
             switch(msg.type) {
                 case DTLS_EVENT_READ:
-                    xtimer_remove(&timeout_timer);
                     return msg.content.value;
                 case DTLS_EVENT_TIMEOUT:
```

Comment:
With the timer removed, we don't need to handle `DTLS_EVENT_TIMEOUT` anymore. Maybe also lose the switch case.